lib/internal/bootstrap/web/exposed-wildcard.js

```diff
@@ -28,16 +28,7 @@ const { URL, URLSearchParams } = require('internal/url');
 exposeInterface(globalThis, 'URL', URL);
 // https://url.spec.whatwg.org/#urlsearchparams
 exposeInterface(globalThis, 'URLSearchParams', URLSearchParams);
-exposeGetterAndSetter(globalThis,
-                      'DOMException',
-                      () => {
-                        const DOMException = lazyDOMExceptionClass();
-                        exposeInterface(globalThis, 'DOMException', DOMException);
-                        return DOMException;
-                      },
-                      (value) => {
-                        exposeInterface(globalThis, 'DOMException', value);
-                      });
+doWhenNotBUildingSnapshot(() => exposeLazyDOMExceptionProperty(globalThis));
```

Comment:
```suggestion
doWhenNotBuildingSnapshot(() => exposeLazyDOMExceptionProperty(globalThis));
```
Looks like a typo, also on other places